test/sql/test_ddl/T/create_table_like

```diff
@@ -0,0 +1,35 @@
+-- name: test_ddl_create_table_like
+create database db_${uuid0};
+use db_${uuid0};
+CREATE TABLE t1 (id1 int, id2 int, id3 double) DUPLICATE KEY(id1) COMMENT "c1" DISTRIBUTED BY HASH(id1) BUCKETS 1 properties('replication_num' = '1');
+CREATE TABLE t2 LIKE t1;
+SHOW CREATE TABLE t2;
+
+CREATE TABLE `test_pk_tbl1` (
+`k1` DATE,
+`k2` DATETIME,
+`k3` VARCHAR(20),
+`k4` VARCHAR(20),
+`k5` BOOLEAN,
+`v1` TINYINT NULL,
+`v2` SMALLINT NULL,
+`v3` INT NULL,
+`v4` BIGINT NOT NULL,
+INDEX init_bitmap_index (k2) USING BITMAP
+) ENGINE=OLAP
+PRIMARY KEY(`k1`, `k2`, `k3`, `k4`, `k5`)
+COMMENT "OLAP"
+PARTITION BY RANGE (k1) (
+    START ("1970-01-01") END ("2030-01-01") EVERY (INTERVAL 30 YEAR)
+)
+DISTRIBUTED BY HASH(`k1`, `k2`, `k3`, `k4`, `k5`) BUCKETS 50
+ORDER BY(`k2`, `v4`)
+PROPERTIES (
+    "replication_num" = "1",
+    "storage_format" = "v2",
+    "enable_persistent_index" = "true",
+    "bloom_filter_columns" = "k1,k2"
+);
+CREATE TABLE test_pk_tbl2 LIKE test_pk_tbl1;
```

Comment:
you need to drop catalog/database/tables

Reply:
done